crates/eips/src/eip1898.rs

```diff
@@ -612,6 +612,9 @@ pub struct BlockNumHash {
 /// Block number and hash of the forked block.
 pub type ForkBlock = BlockNumHash;
 
+/// Either a hash _or_ a number
+pub type NumHash = BlockNumHash;
```

Comment:
maybe should do the opposite, rename the structs and have the aliases be for Block*

Reply:
yeah, but didn't want to rename, although this should be a non-breaking operation

Reply:
I would prefer we did